R/polars_options.R

```diff
@@ -1,3 +1,4 @@
+# TODO: add engine_affinity
```

Comment:
What is engine affinity?

Reply:
pola-rs/polars#20717

In Python, it is specified by an environment variable, but in R, I think it might be useful to specify it by options. This is because Python and R have different engines that can be used, so if we specify an environment variable that is valid in Python, it may not work in R.

Reply:
Thanks